main/src/main/scala/sbt/Defaults.scala

```diff
@@ -513,7 +513,8 @@ object Defaults extends BuildCommon {
     Seq(
       initialCommands :== "",
       cleanupCommands :== "",
-      asciiGraphWidth :== 40
+      asciiGraphWidth :== 40,
+      insideCI :== sys.env.get("BUILD_NUMBER").isDefined || sys.env.get("CI").isDefined
```

Comment:
`sys.env.contains` maybe?

Reply:
So should we wait for `contains` change, append to this PR ourself, or merge as is?